lib/internal/modules/esm/assets/load.js

```diff
@@ -0,0 +1,241 @@
+'use strict';
+const {
+  globalThis,
+  ArrayFrom,
+  ArrayPrototypeJoin,
+  ArrayPrototypePush,
+  ArrayPrototypeSplice,
+  Promise,
+  PromiseResolve,
+  SafeMap,
+  StringPrototypeSlice,
+  StringPrototypePadStart,
+  StringPrototypeSplit,
+  StringPrototypeStartsWith,
+} = primordials;
+const {
+  Buffer: {
+    concat: BufferConcat
+  }
+} = globalThis;
+const {
+  codes: {
+    ERR_UNSUPPORTED_ESM_NETWORK_IMPORT_SPECIFIER,
+    ERR_UNSUPPORTED_ESM_NETWORK_BAD_RESPONSE
+  }
+} = require('internal/errors');
+const { URL } = require('internal/url');
+
+
+/**
+ * Lazily load these to avoid snapshot issues
+ */
+
+function HTTPSGet(url, opts) {
```

Comment:
This should use an http agent.

Reply:
do you have a preferred set of options to default to?

Reply:
Just enabling `keepAlive: true` is enough.

Reply:
done